gourmet/check_encodings.py

```diff
@@ -28,7 +28,7 @@ def __init__ (self, file, encodings=None):
             self.encodings.extend(['utf_16','utf_16_le','utf_16_be'])
         if encodings: self.encodings = encodings
         if type(file) in [str,str]:
```

Comment:
can you fix this while you're at it? This once was `in [str, unicode]` and `unicode` became `str` through `2to3`.